pkg/scheduler/api/job_info.go

```diff
@@ -120,6 +120,7 @@ type TaskInfo struct {
 	VolumeReady bool
 	Preemptable bool
 	BestEffort  bool
+	Elastic     bool
```

Comment:
Why not resure `Preemptable` filed?

Reply:
Preemptable `volcano.sh/preemptable` had  already be occupied by tdm plugin. and its default value is false. so  it can not be used directly.

Reply:
IMO, we can set the elastic pods to be preemptable :)

Reply:
> Preemptable volcano.sh/preemptable had already be occupied by tdm plugin. and its default value is false. so it can not be used directly.

Preemptable is not only used for TDM plugin, we'd like to make annotation more genral for all plugins :)

Reply:
@k82cn In preempt/relclaim case, we want the default value of Preemptable is true, in tdm case,  the default value of Preemptable is false, How do we deal with this difference？

Reply:
preemptable default value is false, which means pod can not be preempted. we can set worker/executor pods `preemptable=true` to describe your workload.

Reply:
@wpeng102 it means that the controller/operator need to create all the worker pod with `preemptable=true` annotation explicitly. it may be better for operator to create the master pod(just one) with `preemptable=false`.

Reply:
Went through this PR last night; I think it's better to ask controller/operator to set `preemptable` explicitly,as in controller/operator, it only few lines of code in PodTempalte with a new annotation :)

Reply:
ok, I changed the code

Reply:
@k82cn but if we do this, it means that preempt/reclaim action will not work unless the operator being ready for the preemptable annotation. it may be a long time

Reply:
> @k82cn but if we do this, it means that preempt/reclaim action will not work unless the operator being ready for the preemptable annotation. it may be a long time

Different operators may have different expectation, it's better to have single behavior in Volcano; let's sync up with other community to align with each other :)